lib/controllers/root-controller.js

```diff
@@ -128,6 +129,7 @@ export default class RootController extends React.Component {
     return (
       <Commands registry={this.props.commandRegistry} target="atom-workspace">
         {devMode && <Command command="github:install-react-dev-tools" callback={this.installReactDevTools} />}
+        <Command command="github:toggle-commit-preview" callback={this.toggleCommitPreviewItem} />
```

Comment:
I think it would be weird to title a pane "View All Staged Changes" or "All Staged Changes."  It's ok for the button and the pane to be labeled differently imo.

Changing the name of the command to match the text that's on the button makes a lot of sense though.  I can take care of that.

Reply:
hmm actually...

"toggle all staged changes" seems less clear than "toggle commit preview."  As a user I'd worry that "toggle all staged changes" is going to unstage the things I have staged, or something like that, rather than just opening a pane.  I vote to leave this as is.